### PR TITLE
Confirm/reject buttons

### PR DESCRIPTION
**Pull request by CressChe**
https://trello.com/c/9fjazLoe/64-confirm-reject-buttons
:)

### CODE REVIEW

File: WhaleSpotting/react-app/src/components/Card.tsx [hunk 1]
```diff
@@ -1,35 +1,61 @@
...
-                    <div>Quantity: {sighting.quantity}</div>
-                    <div>Reported By: {sighting.username} </div>
-                </div>
-                <div data-testid="second-column" className={closeCard ? "second-column closed" : "second-column open"}>
-                    <div>Orca type: {sighting.orcaType}</div>
-                    <div>Orca pod: {sighting.orcaPod}</div>
-                    <div>Longitude: {sighting.longitude} </div>
-                    <div>Latitude: {sighting.latitude} </div>
-                    <div>Description: {sighting.description} </div>
+        <div className={checked ? "hidden" : "" } data-testid="sighting-card">
```
Comment (oskarwilliams) [inline WhaleSpotting/react-app/src/components/Card.tsx]:
```suggestion
        <div hidden={checked} data-testid="sighting-card">
```

File: WhaleSpotting/react-app/src/styles/Card.scss [hunk 2]
```diff
@@ -23,6 +23,18 @@
...
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    flex: 1 1;
+    align-items: center;
+}
+
+.hidden {
+    display: none;
+}
```
Comment (oskarwilliams) [inline WhaleSpotting/react-app/src/styles/Card.scss]: If we do the hidden comment in card we don't need this

**Review by oskarwilliams — CHANGES_REQUESTED** [review]

File: WhaleSpotting/react-app/src/components/Card.tsx (hunk 1 above)
Comment (CressChe) [reply WhaleSpotting/react-app/src/components/Card.tsx]: :)

File: WhaleSpotting/react-app/src/styles/Card.scss (hunk 2 above)
Comment (CressChe) [reply WhaleSpotting/react-app/src/styles/Card.scss]: :)

**Review by oskarwilliams — APPROVED** [review]

**Pull request merged**